pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -461,6 +462,20 @@ func TestDNS(t *testing.T) {
 		t.Fatalf("Received: %v, Expected: %v", event, expectedEvent)
 	}
 
+	// check dns response is traced
+	result = <-ch
+	event = dnstypes.Event{}
+	if err := json.Unmarshal([]byte(result), &event); err != nil {
+		t.Fatalf("failed to unmarshal json: %s", err)
+	}
+
+	expectedEvent.PktType = "HOST"
+	expectedEvent.Qr = "R"
```

Comment:
I think this should at least verify the nameserver as, unlike queries, it is read from a different field for reponses.

Reply:
It is verifying that: I reuse the `expectedEvent` variable which is initialized that way:
```
	// select a nameserver that is not symmetrical with endianness
	nameserver := "8.8.4.4"
...
	expectedEvent = dnstypes.Event{
...
		Nameserver: nameserver,

```

Reply:
Ops, I missed that.